contrib/transaction_profiling_analyzer/transaction_profiling_analyzer.py

```diff
@@ -933,37 +948,39 @@ def print_range_boundaries(range_boundaries, context):
 
             if not omit:
                 if omit_start is not None:
-                    if omit_start == idx-1:
+                    if omit_start == idx - 1:
                         print(" %d. Omitted\n" % (idx))
                     else:
-                        print(" %d - %d. Omitted\n" % (omit_start+1, idx))
+                        print(" %d - %d. Omitted\n" % (omit_start + 1, idx))
                     omit_start = None
 
                 if total_count is None:
                     count_str = '%d sampled %s' % (start_count, context)
                 else:
                     count_str = '%d sampled %s (%d intersecting)' % (start_count, context, total_count)
                 if not shard_count:
-                    print(" %d. [%s, %s]\n     %d sampled %s\n" % (idx+1, start, end, count, context))
+                    print(" %d. [%s, %s]\n     %d sampled %s\n" % (idx + 1, start, end, total_count, context))
```

Comment:
I'm not 100% certain on the intent here, but my best guess for what this should do is the following:

```suggestion
                    print(" %d. [%s, %s]\n     %s\n" % (idx + 1, start, end, count_str))
```